torchok/callbacks/freeze_unfreeze.py

```diff
@@ -56,22 +56,27 @@ def __init__(self, freeze_modules: List[Dict[str, Any]], top_down_freeze_order:
         """Init FreezeUnfreeze.
 
         Args:
-            freeze_modules: List with dictionaries of models to be free-unfreeze.
```

Comment:
Remove Returns: None in `freeze` method please, or add Returns: None to `make_trainable`

Reply:
Removed